backend/src/state_dump.rs

```diff
@@ -28,6 +29,12 @@ impl InMemoryStats {
     }
 }
 
+#[derive(Eq, PartialEq, Ord, PartialOrd, Serialize, Deserialize)]
+pub struct PublicGameInfo {
+    name: String,
+    num_players: usize,
```

Comment:
instead of deriving PartialOrd and Ord here (which will sort by default in ascending order of name, then ascending order of number of players), consider using `sort_by_key` in the sort call?